db/seeds/user_groups.seeds.rb

```diff
@@ -1,6 +1,114 @@
 # frozen_string_literal: true
 
-after :groups_metadata_board, :groups_metadata_councils, :groups_metadata_teams_committees do
+after :groups_metadata_board, :groups_metadata_councils, :groups_metadata_teams_committees, :groups_metadata_delegate_regions do
+  # Delegate Regions
+  UserGroup.create!(
+    name: 'Africa',
+    group_type: :delegate_regions,
+    is_active: true,
+    is_hidden: false,
+    metadata: GroupsMetadataDelegateRegions.find_by!(friendly_id: 'africa'),
+  )
+  UserGroup.create!(
+    name: 'Asia',
+    group_type: :delegate_regions,
+    is_active: true,
+    is_hidden: false,
+    metadata: GroupsMetadataDelegateRegions.find_by!(friendly_id: 'asia'),
+  )
+  UserGroup.create!(
+    name: 'Europe',
+    group_type: :delegate_regions,
+    is_active: true,
+    is_hidden: false,
+    metadata: GroupsMetadataDelegateRegions.find_by!(friendly_id: 'europe'),
+  )
+  UserGroup.create!(
+    name: 'Oceania',
+    group_type: :delegate_regions,
+    is_active: true,
+    is_hidden: false,
+    metadata: GroupsMetadataDelegateRegions.find_by!(friendly_id: 'oceania'),
+  )
+  UserGroup.create!(
+    name: 'Americas',
+    group_type: :delegate_regions,
+    is_active: true,
+    is_hidden: false,
+    metadata: GroupsMetadataDelegateRegions.find_by!(friendly_id: 'americas'),
+  )
+  UserGroup.create!(
+    name: 'North America',
+    group_type: :delegate_regions,
+    is_active: false,
+    is_hidden: false,
+    metadata: GroupsMetadataDelegateRegions.find_by!(friendly_id: 'north-america'),
+  )
+  UserGroup.create!(
+    name: 'South America',
+    group_type: :delegate_regions,
+    is_active: false,
+    is_hidden: false,
+    metadata: GroupsMetadataDelegateRegions.find_by!(friendly_id: 'south-america'),
+  )
+  UserGroup.create!(
+    name: 'Asia East',
+    parent_group_id: GroupsMetadataDelegateRegions.find_by!(friendly_id: 'asia').user_group.id,
```

Comment:
`create` has a return value: It returns the entity you just created. So when calling the create statement for `asia` above, you can store that in a variable and use it here, instead of re-reading the thing you literally just wrote to the DB a few seconds ago

Reply:
(same logic applies to all other child groups below as well)

Reply:
Done